app/controllers/conversions/projects_controller.rb

```diff
@@ -22,6 +22,37 @@ def create
     end
   end
 
+  def edit
+    @project = Project.find(params[:id])
+    authorize @project
+
+    @project = Conversion::EditProjectSharepointLinkForm.new(project: project)
+    @project_id = params[:id]
+  end
+
+  def update
+    authorize project
+    @project = Conversion::EditProjectSharepointLinkForm.new(project: project, args: sharepoint_link_params)
+    @project_id = params[:id]
```

Comment:
You don't need to pass the @project_id if you change the url in the form from `conversions_update_path(@project_id)` to `conversions_update_path(@project)`